### PR TITLE
0.1.2 patch release

### PR DESCRIPTION
**Pull request by nzjony**
Release to update the website: https://heremaps.github.io/harp-map-editor/

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -126,9 +126,9 @@
     "@here/harp-datasource-protocol" "^0.17.0"
     "@here/harp-utils" "^0.17.0"
 
-"@here/harp-omv-datasource@~0.17.0":
+"@here/harp-vectortile-datasource@~0.17.0":
```
Comment (ninok) [inline yarn.lock:129]: Why is this one still on 0.17?

Comment (nzjony) [reply yarn.lock:129]: Good question....

File: src/map-editor/map-handler/index.ts [hunk 2]
```diff
@@ -175,14 +175,14 @@ class MapHandler extends EventEmitter {
...
+            this.m_datasource = new VectorTileDataSource({
                 baseUrl: "https://xyz.api.here.com/tiles/herebase.02",
                 apiFormat: APIFormat.XYZOMV,
                 styleSetName: style || undefined,
-                maxZoomLevel: 17,
+                maxDisplayLevel: 17,
                 authenticationCode: accessToken,
                 copyrightInfo: this.m_copyrights,
-                decoder: new OmvTileDecoder(),
+                decoder: new VectorTileDecoder(),
```
Comment (ninok) [inline src/map-editor/map-handler/index.ts:185]: Oh, we don't use web-workers in the editor ?

File: package.json [hunk 3]
```diff
@@ -55,7 +54,7 @@
         "react-json-tree": "^0.11.2",
         "sass-loader": "^8.0.0",
         "style-loader": "^1.0.0",
-        "three": "^0.117.1",
+        "three": "^0.119.0",
```
Comment (ninok) [inline package.json:57]: 119.1 ?

Comment (nzjony) [reply package.json:57]: I checked the peerDependencies property in the package.json, 0.119.0 is correct

**Review by ninok — APPROVED** [review]

**Pull request merged**